flask_flatpages/page.py

```diff
@@ -7,13 +7,11 @@
 
 """
 
-import yaml
-
 from werkzeug.utils import cached_property
+import yaml
```

Comment:
I'll introduce `flake8-import-order` to the project with `smarkets` config by default.